Added a bunch of new endpoints, all of them linked to club activities

Sigeth opened the pull request:
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
New endpoints supported for the live services part


* **What is the current behavior?** (You can also link to an open issue here)
/


* **What is the new behavior (if this is a feature change)?**
Added :
getClubCampaignById
getClubRoomById
getClubSkins
getClubBucketById
getClubById
getClubActivitiesById
Tests for all of these functions

* **Other information**:
Now it's better than the last time xD
Hope it's all good !

Sigeth committed:
Created getClubCampaignById

Sigeth committed:
Updated getClubCampaignById desc

Sigeth committed:
Added getClubRoomById

Sigeth committed:
Moved getClubRoomById to make more sense

Sigeth committed:
Added getClubSkins

Sigeth committed:
Updated small descs

Sigeth committed:
Added getClubSkinById

Sigeth committed:
Added getClubCompetitions

Sigeth committed:
Added getClubById

Sigeth committed:
Added getClubActivitiesById

Sigeth commented:
Is there anything that I need to change ?

breeku commented:
> 
> 
> Is there anything that I need to change ?

Hey, sorry it's taking time to answer...

By quick glance it look's good! I will have to test the functions myself before we merge. I will try to get it done this week

Sigeth commented:
If you want, I can change the .spec.ts file so you can gain some time, and just test it more easily, I just have to figure out how the format works
And don't worry about the time :D

Sigeth committed:
Changed getClubSkinById to getClubBucketById

I realised that "buckets" are used to store items, so it can leads to a skin-upload, but also a map-upload or an item-upload

Sigeth committed:
Added some tests

Sigeth commented:
So I've done the tests, you can use it if it's good :D

Sigeth commented:
🆙

Sigeth commented:
🆙 🆙 🆙 😄

voblivion commented on src/lib/api/live-services.ts:
```diff
@@ -388,6 +527,111 @@ export const getClubs = async (
+}
+
+/**
+ * This is used to obtain the competitions in the Events section of the game
```
Copy-paste mistake "competitions" -> "activities"
